.eslintrc.json

```diff
@@ -44,6 +43,12 @@
     "@typescript-eslint/restrict-plus-operands": "off",
     "@typescript-eslint/no-floating-promises": "off",
     "@typescript-eslint/prefer-regexp-exec": "off",
+
+    // Added by Ben while upgrading node
+    "@typescript-eslint/no-unsafe-argument": "off",
+    "@typescript-eslint/no-unsafe-enum-comparison": "off",
+    "@typescript-eslint/no-misused-promises": "off",
```

Comment:
Just by looking at the name of these tracks, they all sound important. What's the rational of disabling?

Reply:
As discussed during standup, I checked which rules would be relatively easy to implement: 

Re-enabling `@typescript-eslint/no-unsafe-enum-comparison` was not too bad so I added that back.

`@typescript-eslint/no-unsafe-argument` disables the use of `any` types as arguments, which we do all over the place, so I left it disabled

`@typescript-eslint/no-misused-promises` checks for multiple conditions, but disallows using async functions as handlers such as many of our `onConfirm`s. Thus, I have kept the rule, but disabled errors for that condition. Here is the [documentation](https://typescript-eslint.io/rules/no-misused-promises/) if interested in checking the other conditions.